python/paddle/trainer_config_helpers/layers.py

```diff
@@ -2802,18 +2939,20 @@ def tensor_layer(input, size, act=None, name=None,
 
     .. code-block:: python
 
-       tensor = tensor_layer(input=[layer1, layer2])
+       tensor = tensor_layer(x1=layer1, x2=layer2, size=1000)
```

Comment:
the input of tensor_layer is x1 and x2, conv_shift_layer is a and b, and the dotmul_operator is x and y, can make them unifined?